errors/CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Features
+
+* [#15989](https://github.com/cosmos/cosmos-sdk/pull/15989) Add `ErrStopIterating` for modules to use for breaking out of iteration.
```

Comment:
sorry to be always nitpicky about the changelog lol.
Error never reached v1.0.0 yet (still in beta tags), so think should be under the feature category under v1.0.0.